heppy/Doc.py

```
@@ -62,7 +62,7 @@ def get_module(self, ns):
         if ns in self.nsmap:
             ns = self.nsmap[ns]
         if self.modules == {}:
-            for name, nsi in self.nsmap.iteritems():
+            for name, nsi in self.nsmap.items():
                 self.modules[nsi] = name
         module = self.modules.get(ns)
         if isinstance(module, basestring):
```

Ruff (0.8.2)

Undefined name `basestring`

(F821)